syfertext/doc.py

```diff
@@ -123,35 +123,30 @@ def __iter__(self):
             # Yield a Token object
             yield self[i]
 
-    def getEncryptedVector(self, *workers, crypto_provider=None, requires_grad=True):
+    def get_vector(self, *workers, crypto_provider=None, requires_grad=True):
         """
-           Create one big vector composed of the concatenated Token vectors included in the
-           Doc. The returned vector is SMPC-encrypted.
-
-           TODO: This method should probably be removed. It served for a prototype test,
-                 but concatenating all token vectors of the Doc into one big vector
-                 might not be really useful for practical usecases.
+           Get the mean of the vectors of each Token in this documents.
+           Note: The token vectors are SMPC-encrypted.
         """
 
```

Comment:
Please also include docstring explaining the arguments and the returned value using google-style docstrings:

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Oh I actually added docstrings in the current style we have in the project.